test/task/SetRemoteDescriptionTask.test.ts

```diff
@@ -129,118 +125,6 @@ describe('SetRemoteDescriptionTask', () => {
     });
   });
 
-  describe('run with Firefox 68', () => {
```

Comment:
Though we remove FF68, will we need to add test for FF75+?

Reply:
I think FF68 has a specific workaround that we removed so no need to cover :)